lib/cmd/pack/get-webpack-config.js

```diff
@@ -0,0 +1,380 @@
+  config.module
+    .rule('vue')
+    .test(/\.vue$/)
+      .use('vue-loader')
+        .loader('vue-loader');
+
+  config.module
+    .rule('babel')
+    .test(/\.js$/)
+      .use('babel-loader')
+        .loader('babel-loader')
+        .options({
+          plugins: [
+            'lodash'
+          ],
+          presets: [
+            ['@babel/env', {
+              targets: helpers.getConfigFileOrBrowsersList('babelTargets'),
+              modules: false
+            }]
+          ]
+        });
+
+    config.module
+      .rule('css')
+      .test(/\.css$/)
+        .use('style-loader')
+          .loader('style-loader')
+          .end()
+        .use('css-loader')
+          .loader('css-loader')
+          .options({
+            importLoaders: 1
+          })
+          .end()
+        .use('postcss-loader')
+          .loader('postcss-loader')
+          .options({
+            postcssOptions: {
+              plugins: [
+                cssImport(),
+                mixins(),
+                nested(),
+                simpleVars(),
+              ]
+            }
+          });
+
+  config.optimization
+    .runtimeChunk('single')
+    .splitChunks({
+      chunks: 'all',
+    })
+    .merge({
+      chunkIds: 'named',
+      moduleIds: 'named',
+    });
+
+  config.merge({
+    resolve: {
+      fallback: {
+        crypto: false,
+        fs: false,
+        path: false,
+        vm: false
+      }
+    }
+  });
+
+  /* eslint-enable indent */
+
+  return config;
+}
+
+/**
+ * If the package consuming Clay CLI sets a customizer function in
+ * `claycli.config.js`, pass it the stateful configuration object. If not,
+ * return the config object unmodified.
+ *
+ * @param {Config} config - The webpack-chain configuration object.
+ * @returns {Config} - The modified configuration object.
+ * @throws {Error} - If the customizer returns an invalid config object.
+ */
+function buildCustomConfig(config) {
+  const customizer = getConfigValue('packConfig');
+
+  if (!customizer) {
+    return config;
+  }
+
+  const customConfig = customizer(config);
+
+  // `toConfig` is always defined on a webpack-chain configuration. A
+  // non-webpack-chain object with a `toConfig` method could still fool us, but
+  // that's not really our problem.
+  if (
+    typeof customConfig !== 'object' ||
+    typeof customConfig.toConfig !== 'function'
+  ) {
+    throw new Error(`Expected packConfig to return a webpack-chain configuration. Got ${ typeof customConfig }.`);
+  }
+
+  return customConfig;
+}
+
+/**
+ * Given a stateful webpack-chain configuration object, set sensible defaults
+ * for a development environment.
+ *
+ * @param {Config} config - The webpack-chain configuration object.
+ * @returns {Config} - The modified configuration object.
+ */
+function buildDevelopmentConfig(config) {
+  config
+    .mode('development')
+    .devtool('cheap-module-source-map');
+
+  config.output
+    .chunkFilename('scripts/[name].js?[contenthash]')
```

Comment:
Assuming this syntax is a webpack-chain thing.

Reply:
The path placeholders are Webpack's [template strings](https://webpack.js.org/configuration/output/#template-strings). `name` is pretty much what it says on the can: component scripts get `component-name.js`, vendor scripts typically get an ID.